tests/components/triggercmd/test_config_flow.py

```diff
@@ -0,0 +1,190 @@
+"""Define tests for the triggercmd config flow."""
+
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from homeassistant.components.triggercmd.const import CONF_TOKEN, DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+
+invalid_token_with_length_100_or_more = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjEyMzQ1Njc4OTBxd2VydHl1aW9wYXNkZiIsImlhdCI6MTcxOTg4MTU4M30.E4T2S4RQfuI2ww74sUkkT-wyTGrV5_VDkgUdae5yo4E"
+invalid_token_id = "1234567890qwertyuiopasdf"
+
+
+@pytest.fixture
+def mock_hub():
+    """Create a mock hub."""
+    with patch("homeassistant.components.triggercmd.hub.Hub") as mock_hub_class:
+        mock_hub_instance = mock_hub_class.return_value
+        mock_hub_instance.test_connection = MagicMock(return_value=True)
+        yield mock_hub_instance
```

Comment:
Please move to `conftest.py`

Reply:
I moved it, but let me know if I should delete conftest.py because I didn't end up using that mock_hub fixture.